app/src/main/java/com/github/polypoly/app/game/GameLobby.kt

```diff
@@ -1,30 +1,28 @@
 package com.github.polypoly.app.game
 
 import com.github.polypoly.app.game.user.User
-import kotlin.time.Duration
 
-class GameLobby(
-    val admin: User,
-    val gameMode: GameMode,
-    val minimumNumberOfPlayers: Int,
-    val maximumNumberOfPlayers: Int,
-    val roundDuration: Duration,
-    val gameMap: List<Zone>,
-    val initialPlayerBalance: Int,
-    val name: String,
-    val code: String,
+data class GameLobby(
+    val admin: User = User(),
+    val gameMode: GameMode = GameMode.LAST_STANDING,
+    val minimumNumberOfPlayers: Int = 2,
```

Comment:
Be careful when merging this because in the new version we have a new class that contains all the rules, `GameRules`

Reply:
Yes this will be fixed when merging the other branch on main